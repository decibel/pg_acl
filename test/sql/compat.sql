\set ECHO none

\i test/pgxntool/setup.sql

-- TODO: remove
SET search_path=public,tap;

SELECT plan(1);

SELECT throws_like(
  $$SELECT '=abcdefghijklmnopqrstuvwxyz'::aclitem$$
  , format( '%% must be one of "%s"', _aclitems_all_rights_no_grant() )
  , 'Ensure _aclitems_all_rights() is correct.'
);

SELECT * FROM finish();

\echo TRANSACTION INTENTIONALLY LEFT OPEN!

-- vi: expandtab sw=2 ts=2
