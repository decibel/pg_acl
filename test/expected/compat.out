\set ECHO none
1..1
ok 1 - Ensure _aclitems_all_rights() is correct.
# TRANSACTION INTENTIONALLY LEFT OPEN!
